src/couch/test/eunit/couchdb_file_compression_tests.erl

```diff
@@ -149,21 +152,10 @@ populate_db(Db, NumDocs) ->
                 ]}
             )
         end,
-        lists:seq(1, 500)
+        lists:seq(1, 100)
```

Comment:
Would this be better easier to read if these constants were factored into a variable named e.g. `BatchSize` or `BatchCount`?

Reply:
Good idea. That would look better. Will update.